src/raftstore/coprocessor/split_observer.rs

```diff
@@ -106,13 +105,21 @@ mod test {
         req
     }
 
-    fn new_row_key(table_id: i64, row_id: i64, column_id: u64, version_id: u64) -> Vec<u8> {
+    fn new_batch_split_request(keys: Vec<Vec<u8>>) -> AdminRequest {
+        let mut req = AdminRequest::new();
+        req.set_cmd_type(AdminCmdType::BatchSplit);
+        for key in keys {
+            let mut split_req = SplitRequest::new();
+            split_req.set_split_key(key);
+            req.mut_splits().mut_requests().push(split_req);
+        }
+        req
+    }
+
+    fn new_row_key(table_id: i64, row_id: i64, version_id: u64) -> Vec<u8> {
         let mut buf = Vec::with_capacity(table::ID_LEN);
         buf.encode_i64(row_id).unwrap();
         let mut key = table::encode_row_key(table_id, &buf);
-        if column_id > 0 {
-            key.write_u64::<BigEndian>(column_id).unwrap();
-        }
         key = encode_bytes(&key);
         key.write_u64::<BigEndian>(version_id).unwrap();
```

Comment:
Could we use `key.encode_u64(version_id).unwrap()` instead?

Reply:
Let's do it in other PR.